src/Nethermind/Nethermind.Network.Test/ForkInfoTests.cs

```diff
@@ -56,8 +56,10 @@ public void Fork_id_and_hash_as_expected(long head, string forkHashHex, long nex
         [TestCase(0, "0xa3f5ab08", 1_561_651L, "Unsynced")]
         [TestCase(1_561_650L, "0xa3f5ab08", 1_561_651L, "Last Constantinople block")]
         [TestCase(1_561_651L, "0xc25efa5c", 4_460_644L, "First Istanbul block")]
-        [TestCase(4_460_644L, "0x757a1c47", 0L, "First Berlin block")]
-        [TestCase(6_000_000L, "0x757a1c47", 0L, "Future Berlin block")]
+        [TestCase(4_460_644L, "0x757a1c47", 5_062_605, "First Berlin block")]
+        [TestCase(4_600_000L, "0x757a1c47", 5_062_605, "Future Berlin block")]
+        [TestCase(5_062_605L, "0xB8C6299D", 0L, "First London block")]
+        [TestCase(5_979_794L, "0xB8C6299D", 0L, "Future London block")]
```

Comment:
For Berlin I confirmed all the fork hashes with the Geth team. Would you do the same with Martin? He responded quickly the last time.